activities.el

```diff
@@ -332,6 +333,26 @@ Kills buffers that have only been shown in that activity's
 frame/tab."
   :type 'boolean)
 
+(defcustom activities-sort-by #'activities-sort-by-active-age
+  "How to sort activities during selection.
+Function used to sort by when prompting for activities.  By
+default, a function is used which sorts active activities first,
+and then by age since modification.  If nil, no sorting will be
+applied.  A custom predicate function may also be set.  It should
+take two arguments, both activity names (strings), and return
+non-nil if the first activity should sort before the second."
+  :type `(choice (const :tag "No sorting" nil)
+		 (const :tag "Active state and age" ,#'activities-sort-by-active-age)
```

Comment:
Rather than `const`, this should probably be `function-item`.  (The details of the customization system are covered in the Elisp manual; there's a lot to it.)

Reply:
OK done.  I dislike how `function-item` and others mention the function name in the custom interface, instead of the `:tag` so that's why I used `const`.  But adding `:doc` is a decent workaround.

Reply:
Hm, that's a good point.  I'll leave this decision up to you, then.  :)

Reply:
I think it's fine with `:doc`, and certainly more semantically meaningful.